times_test.go

```diff
@@ -37,6 +37,15 @@ func TestPast(t *testing.T) {
 	}.validate(t)
 }
 
+func TestReltimeOffbyone(t *testing.T) {
+	testList{
+		{"a", RelTime(time.Unix(0, 0), time.Unix(7*24*60*60, 0), "ago", ""), "1 week ago"},
+		{"b", RelTime(time.Unix(0, 0), time.Unix(7*24*60*60, 1), "ago", ""), "1 week ago"},
+		{"c", RelTime(time.Unix(0, 0), time.Unix(14*24*60*60, 0), "ago", ""), "2 weeks ago"},
+		{"d", RelTime(time.Unix(0, 0), time.Unix(14*24*60*60, 1), "ago", ""), "2 weeks ago"},
```

Comment:
Can you add two more test cases, `time.Unix(7*24*60*60, -1)` and `time.Unix(14*24*60*60, -1)`, for complete coverage of the boundary?

Reply:
Note that it says  "6 days" one nanosecond before a week.